server/Main.hs

```diff
@@ -7,12 +7,12 @@
 module Main (main) where
 
 import Network.Wai.Handler.Warp
-import Data.IORef
-import Data.Time.Clock
+import Data.IntMap.Strict (empty)
```

Comment:
Style-wise, it may be better to import this as qualified so that the use of `empty` further down is clearer, e.g.
```suggestion
import qualified Data.IntMap.Strict as IM
```
here and then `IM.empty` later?

Reply:
Done with https://github.com/fpclass/perpetual-haskelling-initiative/pull/20/commits/023065779ab8329cee0c1fc83184d461ddf17437